airbyte-integrations/bases/base-java/build.gradle

```diff
@@ -7,7 +7,6 @@ dependencies {
     implementation 'commons-cli:commons-cli:1.4'
 
     implementation project(':airbyte-protocol:models')
-    implementation project(':airbyte-queue')
```

Comment:
Removing this from https://github.com/airbytehq/airbyte/pull/4699/files#r672182180